app/javascript/gobierto_data/modules/gobierto_data_controller.js

```diff
@@ -3,6 +3,7 @@ import { router } from "../lib/router";
 import 'codemirror/lib/codemirror.css'
 
 Vue.config.productionTip = false;
+Vue.config.devtools = true;
```

Comment:
Is it dangerous for production? (less performance?)

Reply:
Currently is enabled in some other modules. But yes, it's not a good practice, the ideal is about to have something like this: https://webpack.js.org/plugins/environment-plugin/, then when just replace that `true` with `process.env.DEBUG`

Reply:
Here we are https://github.com/PopulateTools/gobierto/issues/3680